syntax = "proto3";
package light;

enum HardwareConfig {
  DELTA = 0;
  BRAVO = 1;
  VANGUARD = 2;
}
